modules/nf-core/bismark/genomepreparation/main.nf

```diff
@@ -20,14 +20,11 @@ process BISMARK_GENOMEPREPARATION {
     script:
     def args = task.ext.args ?: ''
     """
-    mkdir BismarkIndex
-    mv $fasta BismarkIndex/
-
     bismark_genome_preparation \\
         $args \\
         BismarkIndex
 
-    rm BismarkIndex/$fasta
+    rm $fasta
```

Comment:
Do you need to remove it? As it is being staged into this subfolder it should be ignored now?

Reply:
I'd rather be sure I don't have it since I'm trying to get this exact folder out of the process.
Otherwise that'll cause issues downstream

Reply:
yes need to make sure that it is removed or there could be a clash in the next align step